### PR TITLE
Feature/data 1358

### PR DESCRIPTION
**Pull request by philribbens-hu**
Fill out schemas

### CODE REVIEW

File: schemas/com.health-union/nexus_global_context/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "recommended_content_ids": {
+      "type": "array"
+    },
+    "recommendation_location": {
+      "type": ["string", "null"]
+    }
+  },
+  "required": [
+    "site_id",
+    "version_id",
```
Comment (JakeBeresford) [inline schemas/com.health-union/nexus_global_context/jsonschema/1-0-0]: version_id and git_commit_id are not currently implemented in nexus (we don't have that data ATM), might want to make those not required for now?

File: schemas/com.health-union/nexus_user_context/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    "user_role": {
+      "type": "string"
+    }  
+  },
+  "required": [
+    "snowplow_user_id",
+    "google_analytics_id",
+    "wordpress_user_id",
+    "facebook_pixel_user_id",
+    "qualtrics_user_id",
```
Comment (JakeBeresford) [inline schemas/com.health-union/nexus_user_context/jsonschema/1-0-0]: Many of these fields are also not yet implemented. Tickets were created in the nexus data epic: https://health-union.atlassian.net/browse/NEXUS-836 but are still open.

These few will definitely not be implemented by launch: Pendo, facebook, liveramp, qualtrics - Krux is a maybe, depending on whether the data aligns with what we need to send to DFP (I believe it does)

File: schemas/com.health-union/nexus_global_context/jsonschema/1-0-0 (hunk 1 above)
Comment (philribbens-hu) [reply schemas/com.health-union/nexus_global_context/jsonschema/1-0-0]: Ok, makes sense. Removed from required list.

File: schemas/com.health-union/nexus_user_context/jsonschema/1-0-0 (hunk 2 above)
Comment (philribbens-hu) [reply schemas/com.health-union/nexus_user_context/jsonschema/1-0-0]: Gotcha, removed those from required list.

**Review by JakeBeresford — APPROVED** [review]
LGTM in terms of implementation vs. schema. Will defer to @Dylan-Health-Union to review the schema structure and any nuances with that.

**Pull request merged**